enhancements/cvo/object-removal-manifest-annotation.md

```diff
@@ -0,0 +1,142 @@
+---
+title: use-manifest-annotation-for-object-removal
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@crawford"
+  - "@sdodson"
+creation-date: 2020-08-03
+last-updated: 2020-08-04
+status: implementable
+---
+
+# Use Manifest Annotation For Object Removal
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Currently CVO managed object removal is handled by jobs.
+Jobs have many immutable components, so they aren't a great match for CVO managed manifests.
+Instead of the current approach manifests will be used by adding a new annotation to a manifest which will be handled by the CVO causing it to delete the in-cluster object rather than creating/reconciling the manifest.
+This will provide a more straightforward way to remove content.
+
+## Motivation
+
+There should be a straightforward way to remove CVO managed objects.
+
+### Goals
+
+* Users will be able to remove any of the currently managed CVO objects by simply modifying the object's exisitng manifest and adding the new delete annotation.
+
+### Non-Goals
+
+* This enhancement does not add any new objects for managment by CVO therefore the new delete annotation only applies to the currently supported manifests and any that may be added in the future.
```

Comment:
nit: "by CVO therefore" -> "by CVO, therefore" (adding a comma)

Reply:
Fixed in commit [262765d](https://github.com/openshift/enhancements/pull/419/commits/262765d0e846f11bb32d17fe4841e5b9917d4a82)